Code/Tools/AssetProcessor/native/utilities/PlatformConfiguration.cpp

```diff
@@ -1241,10 +1241,13 @@ namespace AssetProcessor
 
                 // New assets can be saved in any scan folder defined except for the engine root.
                 const bool canSaveNewAssets = !isEngineRoot;
+
+                QString watchFolderPath = QString::fromUtf8(scanFolderEntry.m_watchPath.c_str());
+                watchFolderPath = AssetUtilities::NormalizeDirectoryPath(watchFolderPath);
                 AddScanFolder(ScanFolderInfo(
-                    QString::fromUtf8(scanFolderEntry.m_watchPath.c_str(), aznumeric_cast<int>(scanFolderEntry.m_watchPath.Native().size())),
-                    QString::fromUtf8(scanFolderEntry.m_scanFolderDisplayName.c_str(), aznumeric_cast<int>(scanFolderEntry.m_scanFolderDisplayName.size())),
-                    QString::fromUtf8(scanFolderEntry.m_scanFolderIdentifier.c_str(), aznumeric_cast<int>(scanFolderEntry.m_scanFolderIdentifier.size())),
+                    watchFolderPath,
+                    QString::fromUtf8(scanFolderEntry.m_scanFolderDisplayName.c_str()),
+                    QString::fromUtf8(scanFolderEntry.m_scanFolderIdentifier.c_str()),
```

Comment:
Removing the size parameter would actually make this code slightly slower, since the string length would have to be recalculated again.

Reply:
ah, nicely spotted.  i'll fix